owncloudApp/src/main/java/com/owncloud/android/ui/preview/PreviewAudioFragment.kt

```diff
@@ -115,6 +120,16 @@ class PreviewAudioFragment : FileFragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+        collectLatestLifecycleFlow(previewAudioViewModel.getCurrentFile()) { currentFile ->
+            if (currentFile != null) {
+                file = currentFile
+                requireActivity().invalidateOptionsMenu()
```

Comment:
Just to confirm: by doing this here, we don't need to change current references to `file` in the `onPrepareOptionsMenu` and `onOptionsItemSelected` methods, right?

Reply:
Yes, that's right. Te value of `currentFile` is in the variable called `file`, so we don't need to change te reference there. This is the same with all previews. 😃